GNUmakefile

```diff
@@ -16,14 +16,13 @@ build: codegen docs
 
 docs:
 	terraform fmt -recursive examples
-	go get github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs
-	go run github.com/hashicorp/terraform-plugin-docs/cmd/tfplugindocs
+	go tool tfplugindocs
 	mv ./docs/data-sources/ip_ranges.md ./docs/data-sources/ip_range.md
 	rm ./docs/data-sources/*s.md
 	mv ./docs/data-sources/ip_range.md ./docs/data-sources/ip_ranges.md
 	find ./docs/resources/*.md -type f -exec node tools/clean-docs.js {} \;
-	find ./docs/resources/workflow_task_*.md -type f -print0 | xargs -0 sed -i '' 's/subcategory:$$/subcategory: Workflow Tasks/g'
```

Comment:
`sed -i ''` works on macOS but not on Linux
`sed -i` works on Linux but not on macOS

Changed to Perl for maximum compability.